packages/python-sat/patches/proper_build.patch

```diff
@@ -1,20 +1,20 @@
 diff --git a/setup.py b/setup.py
-index 4e810df..d191d6d 100644
+index 82dac9c..3fe102f 100644
 --- a/setup.py
 +++ b/setup.py
-@@ -68,8 +68,8 @@ Details can be found at `https://pysathq.github.io <https://pysathq.github.io>`_
- 
+@@ -69,8 +69,8 @@ Details can be found at `https://pysathq.github.io <https://pysathq.github.io>`_
  # solvers to install
  #==============================================================================
--to_install = ['cadical', 'glucose30', 'glucose41', 'lingeling', 'maplechrono',
--        'maplecm', 'maplesat', 'minicard', 'minisat22', 'minisatgh']
-+to_install = ['cadical', 'glucose30', 'glucose41', 'maplechrono', 'maplecm',
-+        'maplesat', 'minicard', 'minisat22', 'minisatgh']
- 
+ to_install = ['cadical', 'gluecard30', 'gluecard41', 'glucose30', 'glucose41',
```

Comment:
Could you add a comment here saying that we are removing the `lingeling` solver? It's a bit hard to tell from looking at the diff. If you know it would also be great to say why (if you don't know, it would be great if you could try putting it back in  and seeing what goes wrong).

Reply:
Well, it does not work because all the solvers are currently built with a C++ compiler and with the c++11 flag. Lingeling is a solver written in C and so the compiler fails. I can try to put it back but then the set of building scripts needs to be changed. Let me try.